pkg/rootless/rootless_linux.go

```diff
@@ -93,8 +93,8 @@ func tryMappingTool(tool string, pid int, hostID int, mappings []idtools.IDMap)
 		Args: args,
 	}
 
-	if err := cmd.Run(); err != nil {
-		return errors.Wrapf(err, "cannot setup namespace using %s", tool)
+	if output, err := cmd.CombinedOutput(); err != nil {
+		return errors.Wrapf(err, "cannot setup namespace using %s: %s", tool, strings.Replace(strings.TrimSpace(string(output)), "\n", ", ", -1))
```

Comment:
So I understand the replace to remove newlines, but why replace with a comma? Is it two lines separate by a newline?

Reply:
what I wanted to achieve is to replace every newline with a comma (except the last one)

Reply:
@giuseppe Would it make more sense to just logrus.Errorf the full output, newlines and all, then return a more generic "error executing %s" error? It the output gets sufficiently large it could get unreadable

Reply:
ok sure, I can do that